src/__tests__/RocketContainer.test.js

```diff
@@ -0,0 +1,46 @@
+import renderer from 'react-test-renderer';
+import { render } from '@testing-library/react';
+import { BrowserRouter } from 'react-router-dom';
+import '@testing-library/jest-dom/extend-expect';
+import { Provider } from 'react-redux';
+import store from '../redux/configureStore';
+import RocketContainer from '../components/RocketContainer';
+
+describe('Rocket page tests', () => {
+  const rocket = {
+    id: 1,
+    rocket_name: 'Rocket',
+    description: 'Lorem ipsum dolor sit amet',
+    flickr_image: 'https://img.example.com',
+    reserved: false,
+  };
+  it('Page renders as expected', () => {
+    const component = renderer.create(
+      <Provider store={store}>
+        <BrowserRouter>
+          <RocketContainer
```

Comment:
- [ ] Please make sure, all tests passing to make your project approvable!


![Screen Shot 2022-06-17 at 4 01 31 PM](https://user-images.githubusercontent.com/69011963/174286469-b9fd7352-4f03-4d66-b139-36d4a4045741.png)